.github/workflows/helm-ci.yml

```
@@ -27,6 +27,14 @@ jobs:
       - name: Lint Yaml
         run: make helm-lint
 
+      - id: "get_github_app_token"
+        name: "get github app token"
```

IMO we should use title case for step names:

```suggestion
        name: "Get Github app token"
```